terra_notebook_utils/drs.py

```diff
@@ -290,6 +294,34 @@ def copy_to_bucket(drs_uri: str,
         dst_url += f"/{dst_key}"
     copy(drs_uri, dst_url, indicator_type, workspace_name, workspace_namespace)
 
+def copy_batch(manifest: List[Dict[str, str]] = None,
+               drs_urls: Iterable[str] = None,
+               dst_pfx: str = None,
```

Comment:
The above three variables (`manifest`, `drs_urls`, and `dst_pfx`) need the `Optional` type.